.github/workflows/sanity-checks.yml

```diff
@@ -0,0 +1,33 @@
+name: Ansible Sanity Check
+on:
+  pull_request:
+
+jobs:
+  sanity:
+    name: Sanity Check
+    strategy:
+      matrix:
+        ansible:
+          # 2.9 fails due to https://github.com/ansible/ansible/issues/68819
+          - stable-2.10
+          - stable-2.15
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          path: ${{ matrix.ansible }}/ansible_collections/redhatci/ocp
+
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.11
+
+      - name: Install ansible-base ${{ matrix.ansible }}
+        run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
+      
+      # Always succeeding sanity test until all errors are fixed
+      - name: Run sanity test
+        run: ansible-test sanity -v --docker --color --coverage --failure-ok
```

Comment:
could we run it twice once on the PR branch and once on the  main branch and compare the number of  ERROR lines to be sure we are always reducing the number?

Reply:
The GHA is now counting the errors and showing them in the summary: https://github.com/redhatci/ansible-collection-redhatci-ocp/actions/runs/6965977130

I could add also the workflow in the merge of the main branch, at least would give us an idea on errors there vs a PR, would that cover what you have in mind?

Reply:
I pushed a commit with a rebased branch to show what I would like to see.